projects/gradientci.md

```diff
@@ -21,7 +21,21 @@ To create a Gradient Project with continuous integration powered by GradientCI a
 
 ### Configure GradientCI Settings
 
-To set up GradientCI, our continuous integration service, include a directory in your GitHub repository called `.ps_project` with a configuration file `config.yaml`. This file _must_ exist on your default branch \(typically `master`\) in GitHub. We do not yet read the configuration from pull-requests or alternative branches.
+To set up GradientCI, our continuous integration service, include a directory in your GitHub repository called `.ps_project` with a configuration file `config.yaml`, example below.
+
+### Building Branches and Tags
+
```

Comment:
1. could you add an introductory sentence here that clearly states that users can build from branches and/or tags? i don't feel clear from what's here that tags are a viable option, and believe it would be good to highlight that at the outset very explicitly
1. could you add a sentence to this section that explicitly states _how_ to build from tags, or what the behavior is that one can expect?

Reply:
this documentation otherwise looks good to me